test/subset-band-name/environment.yaml

```diff
@@ -0,0 +1,16 @@
+name: papermill-subset-band-name
+channels:
+  - conda-forge
+  - defaults
```

Comment:
does this work if you make this nodefaults?
```suggestion
  - nodefaults
```

Reply:
Worked and pushed.